core/src/components/datetime/test/a11y/datetime.spec.ts

```diff
@@ -0,0 +1,52 @@
+import type { SpecPage } from '@stencil/core/testing';
+import { newSpecPage } from '@stencil/core/testing';
```

Comment:
Was this written as a Stencil spec test vs a Playwright test as a workaround for something?

Reply:
This was intentionally written as a unit test instead of an integration test. Similar tests for other components are spec tests as well (e.g.: [`ion-card`](https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/card/test/aria.spec.ts)). A spec test is more efficient to use here, since we can validate the behavior with JSDOM vs. setting up the entire browser instance.